taskt/UI/Forms/ScriptBuilder Forms/frmScriptBuilder.cs

```diff
@@ -0,0 +1,533 @@
+                tlpControls.RowStyles[1].Height = 0;
+
+                tlpControls.RowStyles[2].SizeType = SizeType.Absolute;
+                tlpControls.RowStyles[2].Height = 81;
+            }
+            else if (_appSettings.ClientSettings.UseSlimActionBar)
+            {
+                tlpControls.RowStyles[2].SizeType = SizeType.Absolute;
+                tlpControls.RowStyles[2].Height = 0;
+            }
+            else
+            {
+                tlpControls.RowStyles[1].SizeType = SizeType.Absolute;
+                tlpControls.RowStyles[1].Height = 0;
+            }
+
+            //get scripts folder
+            var rpaScriptsFolder = Folders.GetFolder(Folders.FolderType.ScriptsFolder);
+
+            if (!Directory.Exists(rpaScriptsFolder))
+            {
+                frmDialog userDialog = new frmDialog("Would you like to create a folder to save your scripts in now? " +
+                    "A script folder is required to save scripts generated with this application. " +
+                    "The new script folder path would be '" + rpaScriptsFolder + "'.", "Unable to locate Script Folder!",
+                    frmDialog.DialogType.YesNo, 0);
+
+                if (userDialog.ShowDialog() == DialogResult.OK)
+                {
+                    Directory.CreateDirectory(rpaScriptsFolder);
+                }
+            }
+
+            //get latest files for recent files list on load
+            GenerateRecentFiles();
+
+            //no height for status bar
+            HideNotificationRow();
+
+            //instantiate for script variables
+            if (!_editMode)
+            {
+                _scriptVariables = new List<ScriptVariable>();
+            }
+            //pnlHeader.BackColor = Color.FromArgb(255, 214, 88);
+
+            //instantiate and populate display icons for commands
+            _uiImages = Images.UIImageList();
+
+            //set image list
+            _selectedTabScriptActions.SmallImageList = _uiImages;
+
+            //set listview column size
+            frmScriptBuilder_SizeChanged(null, null);
+
+            var groupedCommands = _automationCommands.GroupBy(f => f.DisplayGroup);
+
+            foreach (var cmd in groupedCommands)
+            {
+                TreeNode newGroup = new TreeNode(cmd.Key);
+
+                foreach (var subcmd in cmd)
+                {
+                    TreeNode subNode = new TreeNode(subcmd.ShortName);
+
+                    if (!subcmd.Command.CustomRendering)
+                    {
+                        subNode.ForeColor = Color.Red;
+                    }
+                    newGroup.Nodes.Add(subNode);
+                }
+
+                tvCommands.Nodes.Add(newGroup);
+            }
+
+            tvCommands.Sort();
+            //tvCommands.ImageList = uiImages;
+
+            //start attended mode if selected
+            if (_appSettings.ClientSettings.StartupMode == "Attended Task Mode")
```

Comment:
I'm curious what impact this mode has on the runtime. Have we explored the other modes?

Reply:
We haven't. Honestly don't really know what it does yet.